cppcache/src/ThinClientLocatorHelper.cpp

```diff
@@ -74,10 +74,15 @@ ThinClientLocatorHelper::ThinClientLocatorHelper(
       m_sniProxyHost(sniProxyHost),
       m_sniProxyPort(sniProxyPort) {
   for (auto&& locatorAddress : locatorAddresses) {
```

Comment:
Same here, use vector copy or iterator range initialization.

Reply:
Got it 👍